geomstats/geometry/hyperbolic.py

```diff
@@ -456,18 +234,30 @@ def _extrinsic_to_half_plane_coordinates(point):
             (1 - point_ball_x2 - point_ball_y**2) / denom))
         return point_half_plane
 
-    def to_coordinates(self, point, to_coords_type='ball'):
+    @staticmethod
+    def change_coordinates_system(point,
```

Comment:
Yesss! Maybe `from` and `to` are explicit enough for the naming of the parameters in this function, since `coordinates_system` is already in the name of the function.